docs/sources/flow/reference/components/otelcol.processor.filter.md

```diff
@@ -0,0 +1,302 @@
+---
+aliases:
+- /docs/grafana-cloud/agent/flow/reference/components/otelcol.processor.filter/
+- /docs/grafana-cloud/monitor-infrastructure/agent/flow/reference/components/otelcol.processor.filter/
+- /docs/grafana-cloud/monitor-infrastructure/integrations/agent/flow/reference/components/otelcol.processor.filter/
+canonical: https://grafana.com/docs/agent/latest/flow/reference/components/otelcol.processor.filter/
+labels:
+  stage: experimental
+title: otelcol.processor.filter
+description: Learn about otelcol.processor.filter
+---
+
+# otelcol.processor.filter
+
+{{< docs/shared lookup="flow/stability/experimental.md" source="agent" version="<AGENT_VERSION>" >}}
+
+`otelcol.processor.filter` accepts and filters telemetry data from other `otelcol`
+components using the [OpenTelemetry Transformation Language (OTTL)][OTTL].
+OTTL statements consist of [OTTL Converter functions][], which act on paths.
+A path is a reference to a telemetry data such as:
+* Resource attributes.
+* Instrumentation scope name.
+* Span attributes.
+If any condition is met, the telemetry data is dropped (each condition is ORed together).
+
+In addition to the [standard OTTL Converter functions][OTTL Converter functions], 
+the following metrics-only functions are used exclusively by the processor:
+* [HasAttrKeyOnDataPoint][]
+* [HasAttrOnDataPoint][]
+
+[OTTL][] statements used in `otelcol.processor.filter` mostly contain constructs such as:
+* [Booleans][OTTL booleans]:
+  * `not true`
+  * `not IsMatch(name, "http_.*")`
+* [Math expressions][OTTL math expressions]:
+  * `1 + 1`
+  * `end_time_unix_nano - start_time_unix_nano`
+  * `sum([1, 2, 3, 4]) + (10 / 1) - 1`
+
+{{% admonition type="note" %}}
+Some characters inside River strings [need to be escaped][river-strings] with a `\` character.
+For example, the OTTL statement `attributes["grpc"] == true` 
+is written in River as `attributes[\"grpc\"] == true`.
+
+[river-strings]: {{< relref "../../config-language/expressions/types_and_values.md/#strings" >}}
+{{% /admonition %}}
```

Comment:
The next release of the agent will support raw strings, so I updated some of the documentation in #5636. I'm happy to update the `otelcol.processor.filter` docs after your PR is merged though. There's no need to mention raw strings in your PR.